docs/crio.conf.5.md

```diff
@@ -43,6 +43,9 @@ CRI-O reads its storage defaults from the containers-storage.conf(5) file locate
 **file_locking_path**="/runc/crio.lock"
   Path to the lock file.
 
+**clear_storage_on_upgrade**=false
+  If set to true, CRI-O will wipe storage after detecting a new version, to allow for an automatic upgrade.
```

Comment:
Are we only talking about containers/storage here and not container images?  Then we should fix this message.